src/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__image {
+  object-fit: cover;
+  max-width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  padding: 0 16px;
+}
```

Comment:
You've already specified paddings for the card block, why did you need to specify them one more time?